### DISCUSSION TITLE
Possibility to read the playback URL from the RFID Card

### DISCUSSION THREAD

**Discussion by patrick-mira-welt-de** [MiczFlor/RPi-Jukebox-RFID#1958] [Ideas]
Hi everyone,
I am fascinated by the possibilities of the Phoniebox. However, I was thinking, if anyone ever tried to save the target URL on the card itself. An application of this would be for example a band, that hands out an RFID Business card which could play a demo tape upon presenting it to the Phoniebox. 
Does anyone know if this feature was ever implemented or tested on some branch? If not, would it generally be possible? 

Patrick

**Comment by s-martin** [comment]
I think in theory it should be possible, i.e. the used standard should support it: https://en.m.wikipedia.org/wiki/MIFARE

But I'm not aware, if anyone implemented anything like that.

**Comment by davidjdewitt** [comment]
This idea has the potential to greatly simplify the use of Spotify with the PhonieBox.  It is easy to write an NFC tag with  the URL of a Spotify playlist using an iPhone (I assume the same is true with most Android phones as well).   If the PhonieBox reads a card with a Spotify URL it would simply play the associated stream - eliminating the tedious process of registering a stream with the current design.   Just an idea
